src/style.css

```
@@ -1 +1,61 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+header {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  align-items: center;
+}
+
+.img__logo {
+  margin-top: 5px;
+}
+
+.nav__item {
+  list-style: none;
+  display: inline-block;
```

Why inline-block? It's redundant, you have the flex wrapper
```suggestion
  display: block;
```